projects/packages/partner/composer.json

```diff
@@ -3,12 +3,14 @@
 	"description": "Support functions for Jetpack hosting partners.",
 	"type": "jetpack-library",
 	"license": "GPL-2.0-or-later",
-	"require": {},
+	"require": {
+		"automattic/jetpack-connection": "^1.36",
+		"automattic/jetpack-options": "^1.14"
+	},
 	"require-dev": {
 		"brain/monkey": "2.6.1",
-		"yoast/phpunit-polyfills": "1.0.3",
+		"yoast/phpunit-polyfills": "1.0.2",
```

Comment:
`"automattic/jetpack-connection": "^1.36"` required us to downgrade `yoast/phpunit-polyfills` to `1.0.2`